crates/hyperqueue/src/dashboard/data/worker_timeline.rs

```diff
@@ -0,0 +1,72 @@
+use crate::WorkerId;
+use std::time::SystemTime;
+use tako::messages::common::WorkerConfiguration;
+use tako::messages::gateway::LostWorkerReason;
+use tako::server::monitoring::{MonitoringEvent, MonitoringEventPayload};
+
+#[derive(Clone)]
+pub struct WorkerConnectionInfo {
+    worker_id: WorkerId,
+    //todo: implement worker info display
+    _worker_config: Box<WorkerConfiguration>,
```

Comment:
Just remove the field since it's unused for now. And it probably won't have to be boxed here.